src/as2fm/scxml_converter/scxml_entries/scxml_root.py

```diff
@@ -395,6 +386,31 @@ def to_plain_scxml_and_declarations(
             )
         return (converted_scxmls, ros_declarations)
 
+    def to_scxml_with_targets(self, events_to_targets: EventsToAutomata) -> None:
+        """
+        For each "ScxmlSend" instance, add the names of the automata receiving the sent event.
+
+        :param events_to_targets: Mapping between the event name and the automata recipients.
+        """
+        for state in self._states:
+            state.add_target_to_sent_events(events_to_targets)
+
+    def to_scxml_with_replaced_strings(self) -> None:
```

Comment:
```suggestion
    def replace_strings_with_integer_arrays(self) -> None:
```

Reply:
Sure, then I try to come up with a similar naming scheme for the `targets_send_addition` functionality as well